MemorialHouse/MHPresentation/MHPresentation/Source/EditBook/View/EditBookViewController.swift

```diff
@@ -0,0 +1,239 @@
+    
+    // MARK: - Initializer
+    init(viewModel: EditBookViewModel) {
+        self.viewModel = viewModel
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    required init?(coder: NSCoder) {
+        self.viewModel = EditBookViewModel()
+        
+        super.init(coder: coder)
+    }
+    
+    // MARK: - LifeCycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setup()
+        configureNavigationBar()
+        configureAddSubView()
+        configureConstraints()
+        configureKeyboard()
+        configureBinding()
+        configureButtonAction()
+    }
+    
+    // MARK: - Setup & Configuration
+    private func setup() {
+        view.backgroundColor = .baseBackground
+        
+        editPageTableView.delegate = self
+        editPageTableView.dataSource = self
+    }
+    private func configureNavigationBar() {
+        navigationController?.navigationBar.isHidden = true
+    }
+    private func configureAddSubView() {
+        // editPageTableView
+        view.addSubview(editPageTableView)
+        
+        // buttonStackView
+        buttonStackView.addArrangedSubview(addImageButton)
+        buttonStackView.addArrangedSubview(addTextButton)
+        buttonStackView.addArrangedSubview(addVideoButton)
+        buttonStackView.addArrangedSubview(addAudioButton)
+        view.addSubview(buttonStackView)
+        
+        // publishButton
+        view.addSubview(publishButton)
+    }
+    private func configureConstraints() {
+        // tableView
+        editPageTableView.setAnchor(
+            top: view.safeAreaLayoutGuide.topAnchor,
+            leading: view.leadingAnchor,
+            bottom: buttonStackView.topAnchor, constantBottom: 10,
+            trailing: view.trailingAnchor
+        )
+        
+        // buttonStackView
+        buttonStackView.setAnchor(
+            leading: editPageTableView.leadingAnchor, constantLeading: 10,
+            height: 40
+        )
+        buttonStackViewBottomConstraint
+        = buttonStackView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: Self.buttonBottomConstant)
+        buttonStackViewBottomConstraint?.isActive = true
+        
+        // publishButton
+        publishButton.setAnchor(
+            bottom: buttonStackView.bottomAnchor,
+            trailing: editPageTableView.trailingAnchor, constantTrailing: 15,
+            width: 55,
+            height: 40
+        )
+    }
+    private func configureKeyboard() {
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(keyboardWillAppear),
+            name: UIResponder.keyboardWillShowNotification,
+            object: nil
+            )
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(keyboardWillHide),
+            name: UIResponder.keyboardWillHideNotification,
+            object: nil
+            )
```

Comment:
P1: NotificationCenter에 addObserver를 해줄경우 해당 VC가 deinit될 때, observing하고 있는 것을 remove해줘야합니다 ! remove코드도 제 editPhotoViewController에 존재하니 확인해보시고 추가해주시면 좋을 것 같습니다 !

Reply:
제가 알기로는 이 방식으로 추가해준 것은 따로 removeObserver를 안해주어도 되는 것으로 압니다! [문서링크](https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver#discussion)
> If your app targets iOS 9.0 and later or macOS 10.11 and later, you do not need to unregister an observer that you created with this function. If you forget or are unable to remove an observer, the system cleans up the next time it would have posted to it.

하지만, 통일성을 위해서 deinit에 넣어놓겠습니다! (혹시 제가 아는 것과 다른 점이 있으면 알려주세용!)

Reply:
전에 다른 멘토님께 remove 자동으로 된다고 들었는데, (자료 찾아보겠습니다)
꼭 remove 명시해줘야 한다는 자료가 있나요 ??

Reply:
옹 저는 remove를 해줘야한다는 것까지만 읽고 그 뒤에 iOS 9 이상부터는 안해도 된다는 부분을 안 읽었던 것 같습니다...ᐟ.ᐟ 그럼 제 코드에서 remove 부분을 삭제하는게 맞는 것 같네용 .ᐟ.ᐟ 정현님 번거롭게 해드려 죄송합니다 ㅠㅠ